### ISSUE TITLE
Update homepage

### ISSUE THREAD

**Issue by jwhitlock** [mdn/sprints#284]
# User story
As a MDN maintainer, I want to replace the "Get Involved" section with "Help improve MDN", to promote the links and improve performance.

# References
* [bug 1450922](https://bugzilla.mozilla.org/show_bug.cgi?id=1450922) - [FrontEnd] Improve homepage layout for hacks and help MDN

# Acceptance criteria
* "Globe" image is removed
* "Get Involved" section is column 2 in desktop width
* Layout still works in mobile widths

# Tasks
* [ ] Approve new designs
* [ ] Implement new homepage

**Comment by schalkneethling** [comment]
@jwhitlock From our dev planning meeting it sounds like @atopal would prefer to not have this one in this sprint, because of the other potential change that is coming up. Should we move it out of Q3S3>

**Comment by jwhitlock** [comment]
I agree, let's go back to the backlog. We can bring it in again if time allows.

**Comment by jwhitlock** [comment]
Actually, backlog looks extreme. Moving to Review/Test, just not in the milestone.

**Comment by schalkneethling** [comment]
Here are some stats post the updates:
https://speedcurve.com/mozilla-add-ons/mdn/deploy/?b=chrome-%40-3g&dl=307548&dp=307365&r=us-west-1&s=85742&u=128695

Looking good.
